src/domains/wallet/pages/CreateWallet/SuccessStep.tsx

```diff
@@ -72,8 +72,7 @@ export const SuccessStep = ({ onClickEditAlias }: { onClickEditAlias: () => void
 								data-testid="CreateWallet__edit-alias"
 								type="button"
 								variant="transparent"
-								className="text-theme-navy-600"
-								sizeClassName="py-0"
+								className="space-x-0 px-0 py-0 text-theme-navy-600"
```

Comment:
there a few additional classes, is this expected?

Reply:
Yes, so I added them to make sure the default styles (`px-5 py-3 space-x-3 text-base`) don't affect this implementation